4.0-PLAN.adoc

```diff
@@ -3,6 +3,8 @@
 The Jakarta Enterprise Beans 4.0 intended scope includes all plans detailed in the link:https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan[Jakarta EE 9 Release Plan], which in essence involves a backwards incompatible namespace change from `javax.ejb` to `jakarta.ejb`.  In addition the Jakarta Enterprise Beans 4.0 Release Plan will involve three changes:
```

Comment:
I gave this comment before.  I don't know what happened to it.

"three changes" should be "five changes".

Reply:
Fixed.  Thanks, Bill.